src/components/AppLayout/AppLayout.tsx

```diff
@@ -43,6 +46,30 @@ export function AppLayout({
           <Text size="lg" align="center">
             This account has been banned and cannot access the site
           </Text>
+
+          {user?.banDetails?.banReason && (
+            <Stack>
+              <DescriptionTable
+                items={[
+                  { label: 'Reason', value: user?.banDetails?.banReason },
+                  user?.banDetails?.bannedReasonDetails
+                    ? {
+                        label: 'Details',
+                        value: (
+                          <RenderHtml
+                            html={user?.banDetails?.bannedReasonDetails}
+                            style={{
+                              fontSize: '14px',
+                            }}
+                          />
+                        ),
+                      }
+                    : undefined,
+                ].filter(isDefined)}
```

Comment:
We can use te `visible` field instead of filtering the array in here, it'll look cleaner
```suggestion
                items={[
                  { label: 'Reason', value: user?.banDetails?.banReason },
                  {
                    label: 'Details',
                    visible: !!user?.banDetails?.bannedReasonDetails,
                    value: (
                      <RenderHtml
                        html={user?.banDetails?.bannedReasonDetails}
                        style={{
                          fontSize: '14px',
                        }}
                      />
                    ),
                  },
                ]
```